projects/observability/src/shared/components/gauge/gauge.component.test.ts

```diff
@@ -38,6 +38,7 @@ describe('Gauge component', () => {
         x: 0,
         y: -30
       },
+      radius: -30,
```

Comment:
negative radius in the test?

Reply:
Jest provides no width and height for the container. Negative radius and origin is due to that but it still tests that component is applying a margin.

Reply:
Got it - using dom element measurer service would allow us to mock the width/height and make that a bit clearer IMO.

Reply:
ok